### DISCUSSION TITLE
problem with speech enhancement for very noisy audio

### DISCUSSION THREAD

**Discussion by Atefeh197** [espnet/espnet#4991] [General]
Hi,

I use the ESPnet speech enhancement model as a pre-processing step for speech recognition. It improves the results in many cases, 
but for very noisy audio, it corrupts the audio file.

Is there any way to know if the speech enhancement model corrupts a noisy audio file? 

I meant is there any confidence score for ESPnet speech enhancement model?

Thank you

**Comment by Emrys365** [comment]
Hi @Atefeh197, this would be an interesting research problem to measure the confidence score of speech enhancement results. Unfortunately, the current ESPnet toolkit does not support such functions.
